content/applications/finance/fiscal_localizations/mexico.rst

```diff
@@ -1053,22 +1021,17 @@ This report can be generated monthly, and a corresponding XML file is created, i
 An additional report is the *Month 13*: a closing balance sheet that shows any adjustments or
 movements made in the accounting to close the year.
 
-To generate this XML document, navigate to :menuselection:`Accounting --> Accounting -->
-Miscellaneous --> Journal Entries`, and create a new document. Here, add all amounts to modify, and
-balance the debit and/or credit of each one.
-
-After this is done, click :guilabel:`Mark as Closing Entry`, and the report found in
-:menuselection:`Accounting --> Reporting --> Mexico --> Trial Balance Month 13`, contains the total
-amount of the year, plus all the additions of the journal entry.
+To generate this XML document, go to :menuselection:`Accounting --> Accounting -->
+Journal Entries`, and create a new entry for all the amounts to be changed, balancing the debit
+and/or credit of each one. When this is done, select the :guilabel:`Month 13 Closing` option
+(located in the :guilabel:`Other Info` tab, in the :guilabel:`Mexico` section).
+The report with the total amount of the year and all the additions of the journal entry can be found
+in :menuselection:`Accounting --> Reporting --> Trial Balance`, selecting :guilabel:`Month 13` in
+the calendar.
 
-The XML file is generated by pressing the :guilabel:`SAT (XML)` button.
-
-.. image:: mexico/mx-reports-trial-balance-13.png
-   :align: center
-   :alt: Trial Balance Month 13 setup.
+The XML file is created by selecting :guilabel:`SAT (XML)` from the :guilabel:`PDF` menu.
```

Comment:
- I would use a numbered list for this procedure since it's quite long and would allow us to remove the transition words (e.g., *When this is done*) making everything neater and more efficient.
- Make sure to mention steps in the order in which they need to be performed (e.g., In the Other Info tab, enable the Month 13 Closing feature)
- I would also rephrase the last sentence based on my suggestion above.

I cannot add a suggestion per se because there's a deleted line but here it is anyways:

To generate it, proceed as follows:
#. Go to :menuselection:`Accounting --> Accounting --> Journal Entries` and create a new entry for all the amounts to be changed, balancing the debit
and/or credit of each one.
#. In the :guilabel:`Other Info` tab, enable the :guilabel:`Month 13 Closing` option
#. Go to :menuselection:`Accounting --> Reporting --> Trial Balance`, click the calendar, and select :guilabel:`Month 13`.
#. Click the :icon:`caret-down` (:guilabel:`down arrow) next to the :guilabel:`PDF` button, and select :guilabel:`SAT (XML)`.